src/targets/gpu/mlir.cpp

```diff
@@ -73,6 +73,7 @@ namespace gpu {
 
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_MLIR);
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MLIR_TUNE_EXHAUSTIVE);
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MLIR_TUNE_LIMIT);
```

Comment:
Need to document this inside https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/docs/dev/env_vars.rst

Reply:
@pfultz2 can you document this new env please